rtgui/profilestore.h

```diff
@@ -208,11 +208,16 @@ class ProfileStore :
         return internalDynamicEntry;
     }
 
-    std::vector<DynamicProfileRule> &getDynamicProfileRules()
+    const std::vector<DynamicProfileRule> &getDynamicProfileRules() const
     {
         return dynamicRules;
     }
 
+    void setDynamicProfileRules(const std::vector<DynamicProfileRule> &r)
```

Comment:
:+1: I was curious if you would go for `void set...(T)` or `void set...(const T&)`, and you went for the latter which I would have also done, though the former would be faster with an r-value-ref (which isn't how it's used here).

If I'm not mistaken, you could get rid of `#include "dynamicprofile.h"` if you PIMPL'ed the `ProfileStore` (that would be much appreciated) or if you turned `std::vector<DynamicProfileRule> dynamicRules` into `std::unique_ptr<std::vector<DynamicProfileRule>> dynamicRules`. There are already so many unnecessary header dependencies in RT, we shouldn't carelessly add more. :wink: